.gitattributes

```diff
@@ -6,3 +6,4 @@ third_party/lzma.js/lzma-full.js -diff
 tests/* linguist-vendored
 third_party/* linguist-vendored
 system/ linguist-vendored
+tests/other/test_emsize.js
```

Comment:
can you add `-diff` and group it with the other .js files above?

Reply:
Done